### ISSUE TITLE
Error: Could not load /github/workspace/node_modules/@css-inline/css-inline-wasm/index_bg.wasm?module

### ISSUE THREAD

**Issue by ekazda** [nuxt-modules/og-image#243]
I attempt to build my nuxt application using our Github actions, but the build started to fail once nuxt-og-image updated from nuxt-og-image@3.0.0-rc.56 to nuxt-og-image@3.0.0-rc.61. Is anyone else having this build issue?

```
[error] [nitro] Error: Could not load /github/workspace/node_modules/@css-inline/css-inline-wasm/index_bg.wasm?module (imported by node_modules/nuxt-og-image/dist/runtime/nitro/og-image/bindings/css-inline/wasm.js): ENOENT: no such file or directory, open '/github/workspace/node_modules/@css-inline/css-inline-wasm/index_bg.wasm?module'


undefined
[error] Could not load /github/workspace/node_modules/@css-inline/css-inline-wasm/index_bg.wasm?module (imported by node_modules/nuxt-og-image/dist/runtime/nitro/og-image/bindings/css-inline/wasm.js): ENOENT: no such file or directory, open '/github/workspace/node_modules/@css-inline/css-inline-wasm/index_bg.wasm?module'
   

[error] Could not load /github/workspace/node_modules/@css-inline/css-inline-wasm/index_bg.wasm?module (imported by node_modules/nuxt-og-image/dist/runtime/nitro/og-image/bindings/css-inline/wasm.js): ENOENT: no such file or directory, open '/github/workspace/node_modules/@css-inline/css-inline-wasm/index_bg.wasm?module'
```

**Comment by harlan-zw** [comment]
Hey, this should be fixed in [3.0.0-rc.64](https://github.com/nuxt-modules/og-image/releases/tag/v3.0.0-rc.64).

**Comment by ekazda** [comment]
@harlan-zw Confirmed fixed with [3.0.0-rc.64](https://github.com/nuxt-modules/og-image/releases/tag/v3.0.0-rc.64).
